src/js/components/Alert.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Alert = ({ type = 'info', heading, text, htmlElement }) => {
+  if (!text) return null
+
+  return (
+    <div className={`usa-alert usa-alert-${type}`}>
+      <div className="usa-alert-body">
+        {heading ? <h3 className="usa-alert-heading">{heading}</h3> : null}
+        <p className="usa-alert-text">{text}</p>
+        {htmlElement ? htmlElement : null} {/* allow for additional html */}
```

Comment:
Instead of passing in the custom `htmlElement` prop, you can just use `props.children` and wrap children in standard html.